build/hcmi/02-getHCMIData.py

```diff
@@ -474,15 +474,15 @@ def get_copy_call(a):
 
         if math.isnan(a):
             return float('nan')
-        
-        a_val = math.log2(float(a)+0.000001) ###this should not be exponent, should be log!!! 2**float(a)
-        if a_val < 0.0: #0.5210507:
+
+        a_val = math.log2(float(a)+0.000001)
```

Comment:
please double check this. we should not be doing log space unless they somehow threw in an exponent. values shoud range between 0 and 2 (or more in some cases)

Reply:
Copy numbers here are all integer values from 0-214 (with log) but primarily centered around 2 which gets converted to 1 and called as diploid.

Here is a distribution of the values (with log) taken from the most recently generated [version](https://figshare.com/account/projects/189342/articles/28038896?file=51224342).

---

**All counts across all samples:**
![image](https://github.com/user-attachments/assets/dd88f660-d850-4378-9e44-24726f9dcd35)


**All counts across all samples for Copy Numbers 0-20:**
![image](https://github.com/user-attachments/assets/7e62edaa-7ebf-4078-a8aa-057196dfcc59)


**Copy Call Counts Table**
| Count        | Copy Call      |
|-----------|-----------|
| 7622238   | amp       |
| 14048     | deep del  |
| 14310695  | diploid   |
| 5030752   | gain      |
| 1673926   | het loss  |